wordpress-mobile/WordPress-Android
# Fix Featured Image issues when leaving the editor

Fix #4766 by making sure feature image field is tested when checking for changes in `hasChanges` routine.

To test:
- Open the editor for a new post, setting the title and body
- Open post settings and set a featured image
- Tap back twice and go back to posts list
- Edit the post again
- Open post settings and confirm the featured image is still set
- ^ repeat the steps above by removing the FI
- Re-add the FI, and publish the post
- Check that FI is set on the post
